ui/packages/consul-ui/package.json

```diff
@@ -101,96 +106,97 @@
     "d3-selection": "^2.0.0",
     "d3-shape": "^2.0.0",
     "dayjs": "^1.9.3",
-    "deepmerge": "^4.2.2",
+    "deepmerge": "^4.3.1",
     "doctoc": "^2.0.0",
-    "ember-array-fns": "^1.4.0",
-    "ember-assign-helper": "^0.3.0",
-    "ember-auto-import": "^2.10.0",
+    "ember-array-fns": "^1.4.2",
+    "ember-assign-helper": "^0.5.1",
+    "ember-auto-import": "^2.11.1",
     "ember-basic-dropdown": "^8.6.2",
-    "ember-can": "^4.2.0",
-    "ember-changeset-validations": "~3.15.2",
-    "ember-cli": "3.28.6",
-    "ember-cli-app-version": "^5.0.0",
-    "ember-cli-babel": "^7.26.10",
-    "ember-cli-code-coverage": "^1.0.0-beta.4",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-deprecation-workflow": "^2.1.0",
-    "ember-cli-flash": "^4.0.0",
-    "ember-cli-htmlbars": "^5.7.2",
+    "ember-can": "^8.0.0",
+    "ember-changeset": "^5.0.0",
+    "ember-changeset-validations": "~5.0.0",
+    "ember-cli": "^4.12.3",
+    "ember-cli-app-version": "^7.0.0",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-code-coverage": "^3.1.0",
+    "ember-cli-dependency-checker": "^3.3.3",
+    "ember-cli-deprecation-workflow": "^3.4.0",
+    "ember-cli-flash": "^6.0.0",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-page-object": "^1.17.11",
+    "ember-cli-page-object": "^2.3.2",
     "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-string-helpers": "^6.1.0",
+    "ember-cli-string-helpers": "^8.0.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cli-yadda": "^0.7.0",
     "ember-collection": "^3.0.0",
-    "ember-compatibility-helpers": "^1.2.5",
+    "ember-compatibility-helpers": "^1.2.7",
     "ember-composable-helpers": "^5.0.0",
-    "ember-data": "~3.28.6",
-    "ember-data-model-fragments": "5.0.0-beta.8",
+    "ember-data": "~4.0.0",
+    "ember-data-model-fragments": "6.0.10",
```

Comment:
I would recommend matching ember-data-model-fragments 6.0.10 with ember-data 4.6.7 - that is the intended pairing.